claim/index.html

```diff
@@ -100,27 +100,25 @@ <h2>System Setup</h2>
   <p>Make sure you have nodejs and npm installed first. On MacOS, please install <a href="https://brew.sh">homebrew</a> and run "<b>brew install node unbound</b>". On debian/ubuntu, you can run "<b>sudo apt-get install nodejs npm build-essential libunbound-dev</b>". If you run other distributions or OSes, you can probably figure this part out.</p>
   <p>Next, install node-gyp: <b>npm install node-gyp</b></p>
   <h2>Download</h2>
-  <p>Download <b>hsd</b>, <b>hs-client</b>, and <b>hs-airdrop</b> from <a href="https://handshake.org/download/">https://handshake.org/download/</a>. If downloaded from github, the directory structure is slightly different (hsd-2.4.0/hsd should be replaced with just hsd in these instructions).</p>
-  <p>Extract hsd, hs-client, and hs-airdrop: <b>tar xvf hs*</b></p>
+  <p>Download <b>hsd</b> and <b>hs-airdrop</b> from <a href="https://handshake.org/download/">https://handshake.org/download/</a>. If downloaded from github, the directory structure is slightly different (hsd-3.0.1/hsd should be replaced with just hsd in these instructions).</p>
```

Comment:
I'd like to not have to update this html page every time we release a new version - can it be worded so "latest release" works instead? Or maybe just "for example..."

Reply:
Updated to a wildcard, so you can proceed with merging as you prefer. My offer is still valid though 😉